CHANGELOG.md

```diff
@@ -1,6 +1,35 @@
 Change Log for libSplash
 ================================================================
 
+Release 1.7.0
+-------------
+**Date:** 2018-01-22
+
+This release adds modernized CMake scripts including CMake config
+packages on install with targets. A new `readAttributeInfo` method
+was added to the `DataCollector` for save reading of types and
+extends of attributes. On reads, the `offset` in `DataContainer` for
```

Comment:
REQUIRED:
typo either extend or extension (noun) or extending (verb form)

Reply:
actually, I wanted to write `extents`